index.css

```diff
@@ -0,0 +1,151 @@
+*{
```

Comment:
You have done well so far, nice job!!
However, I would recommend that you set up the linters file to ensure your code is well formatted to best practice standards

Reply:
This is really a good review , keep it up